packages/validation-manager/src/ValidationManager.ts

```diff
@@ -288,7 +340,23 @@ export class ValidationManager implements IValidationManager {
       }
     } else {
       // NOTE: this mode doesn't do any opcode checking and no stake checking!
-      res = await this._callSimulateValidation(userOp)
+      // can't decode validationResult at all. we only have "pass-or-not" result
```

Comment:
Do we still need the "unsafe" mode? The entire reason we keep this `bundler` repo maintained is as a test stub for the validation rules, isn't that so?

Reply:
unsafe was for internal testing. and actually, now with "handleOps" validation, unsafe can't be fully implemented (one of the reason to use simulateValidation was to expose inner fields that are only accessible via tracing).
I suggest removing it in a separte PR.

Reply:
We should also remove the js tracer while we're at it.. in a separate pr

Reply:
Created issue:
https://opengsn.atlassian.net/browse/AA-538